src/app/public/modules/auth-http-client/auth-interceptor-default-permission-scope.ts

```diff
@@ -0,0 +1,7 @@
+//#region imports
+
+import { InjectionToken } from '@angular/core';
```

Comment:
```
import {
  InjectionToken
} from '@angular/core';
```
Please and thank you!

Reply:
Sorry about that. Fixed. IntelliJ doesn't have an option for that style so I may miss a few from time to time.